MaterialLib/Fluid/WaterVaporProperties/WaterVaporProperties.h

```diff
@@ -16,67 +16,66 @@
 #include <string>
 #include "MaterialLib/PhysicalConstant.h"
 
-using MaterialLib::PhysicalConstant::MolarMass::Water;
-using MaterialLib::PhysicalConstant::MolarMass::Air;
-using MaterialLib::PhysicalConstant::IdealGasConstant;
 namespace MaterialLib
 {
 namespace Fluid
 {
-/*
+    using MaterialLib::PhysicalConstant::MolarMass::Water;
```

Comment:
This is not allowed in the header file in a namespace, because the MaterialLib::Fluid::Water will always point to MaterialLib::PhysicalConstants::MolarMass::Water.